apps/api/src/app/events/usecases/send-message/send-message.base.ts

```diff
@@ -21,7 +21,7 @@ export abstract class SendMessageBase extends SendMessageType {
     protected createLogUsecase: CreateLog,
     protected createExecutionDetails: CreateExecutionDetails,
     protected subscriberRepository: SubscriberRepository,
-    protected getDecryptedIntegrationsUsecase?: GetDecryptedIntegrations
+    protected getDecryptedIntegrationsUsecase: GetDecryptedIntegrations
```

Comment:
this should not be optional as we have the code that uses this use-case

Reply:
CC. @djabarovgeorge

Reply:
That's true, I made him optional only because at the moment in-app does not require integration.